tests/order_modifiers_integration/Checkout/Fees_Test.php

```diff
@@ -273,6 +290,19 @@ public function it_should_calculate_fees_and_store_them_correctly_complex_math()
 		$cart_total    = $cart->get_cart_total();
 		$cart_subtotal = $cart->get_cart_subtotal();
 
+		$this->set_fn_return( 'wp_create_nonce', '1029384756' );
+		$this->assertMatchesHtmlSnapshot(
+			preg_replace(
```

Comment:
Why a `preg_replace` here? If you're using the AUTOINCREMENT setting, you should be able to use a plain `str_replace`.

Reply:
the preg actually removes styles and scripts coming along with the shortcode. I dont want to test that, its being tests elsewhere. i am only interested in the math and the format here